spacepy/pybats/__init__.py

```diff
@@ -877,49 +1093,249 @@ class IdlFile(PbData):
+    classes that do more preprocessing of the data before returning the
     object.  Hence, using this class to read binary files is typically not
     the most efficient way to proceed.  Look for a PyBats sub module that suits
     your specific needs, or use this base object to write your own.
 
-    A note on byte-swapping: PyBats assumes little endian byte ordering because
+    Multi-Frame Files
+    -----------------
+
+    Typically, Idl-formatted data has a single *frame*, or a single snapshot
+    worth of data (a `*.out` file).  However, it is possible to (externally)
+    combine many of these files together such that a time series of data frames
+    are contained within a single file (`*.outs` files). This class can read
+    these files, but only one data frame can be made available at a time.  This
+    prevents very large memory requirements.
+
+    These files are opened and handled similarly to regular IDL-formatted
+    files  with some important differences.  Upon instantiation, the user
+    may select which data frame to open with the *iframe* kwarg.  The default
+    action is to open the first frame.  The user may learn more about the
+    number of frames within the file and their associated epoch/iteration
+    information by examining the top level *attrs* (see below.)
+    The user may switch to any arbitrary frame using the `switch_frame(iframe)`
+    object method.  This will load the relevant data from disk into the
+    object, overwriting the previous contents.
+
+    If the user has created any new variables using the data from a certain
+    data frame, those new values will not be updated automatically.  An
+    exception is for any *self.calc_* type methods that are set to update
+    automatically.
+
+    Top Level Attributes
+    --------------------
+    Critical file information can be found via the top-level object attributes
+    (e.g., accessing the dictionary `self.attrs`).  All IdlFile objects and
+    child classes begin with at least these attributes:
+
+    | Attribute Name       | Description                                        |
+    | -------------------- | -------------------------------------------------- |
+    | file                 | Path/name of file represented by object            |
+    | iter/time/runtime    | Iteration/datetime/runtime of the current frame    |
+    | iters/times/runtimes | Lists of all iterations/times of each data frame   |
+    | *_range              | The range of iterations/epochs covered in the file |
+    | ndim                 | Number of spatial dimensions covered by the data   |
+    | nframe               | The total number of data frames within the file    |
+    | iframe               | The current frame loaded (zero-based)              |
+    | format               | The format of the file, either binary or ascii     |
+    | header               | The raw string header of the file                  |
+
+    Notes
+    -----
+    
```

Comment:
Do blank lines here affect how the docs render? I'm not sure, but would probably remove it.

Reply:
Not sure, but removing whitespace to be consistent.  Same goes with suggestions below.